src/audio/detect_test.c

```diff
@@ -121,9 +120,9 @@ static void notify_kpb(const struct comp_dev *dev)
 	cd->client_data.sink = NULL;
 	cd->client_data.id = 0; /**< TODO: acquire proper id from kpb */
 	/* time in milliseconds */
-	cd->client_data.history_depth = (cd->history_depth != 0) ?
-					 cd->history_depth :
-					 cd->config.history_depth;
+	cd->client_data.drain_req = (cd->drain_req != 0) ?
+					 cd->drain_req :
+					 cd->config.drain_req;
```

Comment:
I don't see how drain_req is clearer than history_depth. No idea what drain_req means and what the concept is here.

Reply:
You have no idea what `drain_req` is? What about "draining request"? When working with validation teams I notice they are confused by "history depth" and somehow "draining request" is more meaningful to people.